src/components/FormInput/FormInput.test.tsx

```diff
@@ -0,0 +1,88 @@
+import userEvent from '@testing-library/user-event';
+import { useState } from 'react';
+import { describe, it, expect, vi } from 'vitest';
+
+import { render, screen } from '../../test/test-utils';
+
+import { FormInput } from './FormInput';
+
+describe('FormInput', () => {
+  const defaultProps = {
```

Comment:
defaultProps contains a single shared mock for onChange (vi.fn()). Reusing the same mock across multiple tests without resetting it can cause cross-test leakage and false positives/failures.

**Sugerencia:**
```typescript
Create fresh mocks per test or reset mocks in a beforeEach. Example:

let onChange: ReturnType<typeof vi.fn>;
beforeEach(() => { onChange = vi.fn(); });
// use onChange in the render for each test

This ensures test isolation.
```